index.js

```diff
@@ -100,10 +135,13 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
+
+newArr = myMap
+
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
+console.log(myMap([1,2,3,4,5], triple)) 
+// => [3,6,9,12,15]
 
 // console.log(myMap([2,4,6,8], triple))   
 // // => [6,12,18,24]
```

Comment:
Where you able to figure these out after the review?

Reply:
Yes it made more sense after the review, i think i just need to spend more time with the callbacks.